ntp-proto/src/peer.rs

```diff
@@ -124,11 +144,10 @@ pub enum AcceptSynchronizationError {
 }
 
 impl Peer {
-    pub fn new(
-        our_id: ReferenceId,
-        peer_id: ReferenceId,
-        current_system_time: NtpTimestamp,
-    ) -> Self {
+    pub fn new(our_id: ReferenceId, peer_id: ReferenceId, local_clock_time: NtpTimestamp) -> Self {
+        // we initialize with the current time so that we're in the correct epoch.
+        let time = local_clock_time;
```

Comment:
the specification does not initialize the time (it is set before it is read), but using `0` caused issues for us right? because the logic turned out to put us in the wrong epoch.